cmd/boulder-ra/main.go

```diff
@@ -19,81 +20,156 @@ import (
 
 const clientName = "RA"
 
+type config struct {
+	RA struct {
+		cmd.ServiceConfig
+		cmd.HostnamePolicyConfig
+
+		RateLimitPoliciesFilename string
+
+		MaxConcurrentRPCServerRequests int64
+
+		MaxContactsPerRegistration int
+
+		// UseIsSafeDomain determines whether to call VA.IsSafeDomain
+		UseIsSafeDomain bool // TODO: remove after va IsSafeDomain deploy
+
+		// The number of times to try a DNS query (that has a temporary error)
+		// before giving up. May be short-circuited by deadlines. A zero value
+		// will be turned into 1.
+		DNSTries int
+
+		VAService *cmd.GRPCClientConfig
+
+		MaxNames     int
+		DoNotForceCN bool
+
+		// Controls behaviour of the RA when asked to create a new authz for
+		// a name/regID that already has a valid authz. False preserves historic
+		// behaviour and ignores the existing authz and creates a new one. True
+		// instructs the RA to reuse the previously created authz in lieu of
+		// creating another.
+		ReuseValidAuthz bool
+	}
+
+	VA struct {
```

Comment:
The RA shouldn't need to include the VA config. This was a mistake I made when adding the CAASERVFAILExceptions. Will fix shortly.